src/components/CreateSingleSelectionInput.jsx

```diff
@@ -0,0 +1,164 @@
+        updatedTextBox[field] = event.target.value;
+        onTextBoxChange(index, updatedTextBox);
+        if (event.target.id === 'question') {
+            event.target.value === '' ? updateInputEmpty(0, 0) : updateInputEmpty(0, 1);
+        }
+    };
+
+    const handleAdd = () => {
+        const inp = [...inputs, []];
+        const inpEmp = [...inputEmpty, true];
+        setInputs(inp);
+        setInputEmpty(inpEmp);
+    };
+
+    const handleDeleteInput = (i) => {
+        const deleteInp = [...inputs];
+        deleteInp.splice(i, 1);
+        setInputs(deleteInp);
+    };
+
+    const handleInputChange = (onChangeValue, i) => {
+        const inputData = [...inputs];
+        inputData[i] = onChangeValue.target.value;
+        setInputs(inputData);
+        if (onChangeValue.target.id === String(i)) {
+            onChangeValue.target.value === '' ? updateInputEmpty(i + 1, 0) : updateInputEmpty(i + 1, 1);
+        }
+    };
+
+    return (
+        <div className="px-0 pb-4 pb-lg-5">
+            <div className="row justify-content-between pb-2 m-0">
+                <div className="col d-flex justify-content-start p-0">
+                    <h1 className="font-century-gothic text-steel-blue fs-3 fw-bold p-0 m-0">Seleção Única</h1>
+                </div>
+                <div className="col d-flex justify-content-end p-0">
+                    <RoundedButton hsl={[190, 46, 70]} icon={iconFile} />
+                    <RoundedButton className="ms-2" hsl={[190, 46, 70]} icon={iconTrash} onClick={onTextBoxRemove} />
+                </div>
+            </div>
+            <div className="row form-check form-switch pb-3 m-0 ms-2">
+                <input className="form-check-input border-0 fs-5 p-0" type="checkbox" role="switch" id="flexSwitchCheckDefault" />
+                <label className="form-check-label font-barlow fw-medium fs-5 p-0" htmlFor="flexSwitchCheckDefault">
+                    Obrigatório
+                </label>
+            </div>
+            <div className="bg-light-grey rounded-4 lh-1 w-100 p-4">
+                <div className="mb-3">
+                    <label htmlFor="question" className="form-label fs-5 fw-medium">
+                        Pergunta
+                    </label>
+                    <input
+                        type="text"
+                        className="form-control bg-transparent border-0 border-bottom border-steel-blue rounded-0 fs-5 lh-1 p-0"
+                        id="question"
+                        aria-describedby="questionHelp"
+                        onChange={(event) => handleTextBoxChange(event, 'question')}
+                    />
+                    {inputEmpty[0] && (
+                        <div id="questionHelp" className="form-text text-danger fs-6 fw-medium">
+                            *Este campo é obrigatório.
+                        </div>
+                    )}
+                </div>
+                <div className="pb-1">
```

Comment:
```suggestion
                <div className="mb-3">
```
- Apenas uma sugestão de solução para o espaçamento diferente dos outros entre o campo "descrição" e o "opção 0".

Reply:
Fica bem melhor mesmo!! ;-)